config/config.go

```diff
@@ -64,6 +64,7 @@ type ListenFrom struct {
 	Protocol                       *string                `json:",omitempty"`
 	TimeoutDuration                *time.Duration         `json:"-"`
 	ServerAcceptDeadline           *time.Duration         `json:"-"`
+	SpanArgumentReplacementRules   []string               `json:",omitempty"`
```

Comment:
Naming nit: `SpanNameReplacementRules` since this is only about span names. It's really not clear what "argument" would refer to (in the mind of a user/customer).